src/pymovements/plotting/_matplotlib.py

```diff
@@ -314,3 +315,53 @@ def _draw_line_data(
     line_collection.set_linewidth(2)
     line = ax.add_collection(line_collection)
     return line
+
+
+def _set_screen_axes(
+    ax: plt.Axes,
+    screen: Screen | None,
+    *,
+    func_name: str,
+) -> None:
+    """Set axes limits and aspect ratio from gaze.experiment.screen, if available.
+
+    Parameters
+    ----------
+    ax : plt.Axes
+        Matplotlib axes object to modify.
+    screen : Screen | None
+        Screen object from a Gaze's Experiment. If None, no changes are made.
+    func_name : str
+        Name of the plotting function, used in error messages.
+
+    Raises
+    ------
+    ValueError
+        If the screen origin is not 'upper left'.
+    ValueError
+        If the screen width or height is not positive.
+    """
+    if screen is None:
+        return
+
+    # If screen has no pixel info, skip silently
+    if screen.width_px is None or screen.height_px is None:
+        return
+
+    if (
+        screen.width_px is None or screen.height_px is None
+        or screen.width_px <= 0 or screen.height_px <= 0
+    ):
+        raise ValueError(
```

Comment:
as discussed, this value error is not necessary as we can assume the `Screen` to be valid.